hypothesis-python/src/hypothesis/internal/conjecture/utils.py

```diff
@@ -265,7 +265,7 @@ def biased_coin(data, p, *, forced=None):
                 # becomes i > falsey.
                 result = i > falsey
 
-            if i > 1:
+            if i > 1:  # pragma: no branch  # unclear why, but always true on python3.8
```

Comment:
I don't think this makes sense? This definitely should not always be true. A biased coin draw from `ConjectureData.foor_buffer([0])` should result in this branch being false.

Reply:
Aaaand, it's the same bytecode-optimisation-breaking-coverage bug; this time with the following `break` instead of `continue`.  Comment updated accordingly.